packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -84,23 +87,51 @@ const Updates = decorate([
     initialState: () => ({
       ...initialProxyState(),
       ...initialRegistrationState(),
+      ...initialChannels(),
       askRegisterAgain: false,
+      xoaReleaseChannels: [],
       showPackagesList: false,
     }),
     effects: {
       async configure() {
-        await xoaUpdater.configure(
-          pick(this.state, [
+        await xoaUpdater.configure({
+          ...pick(this.state, [
             'proxyHost',
             'proxyPassword',
             'proxyPort',
             'proxyUser',
-          ])
-        )
+          ]),
+          channel: this.state.addPrivateChannel
+            ? this.state.privateChannelId
+            : this.state.channelId,
+        })
         return this.effects.resetProxyConfig()
       },
-      initialize() {
-        return this.effects.update()
+      async initialize() {
+        this.effects.update()
```

Comment:
Never ignore a promise, either `await` it or handle a possible error.

You can use `promise-toolbox/ignoreError` for this:

```js
ignoreError.call(this.effects.update)
```